yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -172,6 +185,36 @@ main() {
         fi
     fi
 
+    last_analyze=$(psql $pg_connection_string -tAqc "SELECT schemaname, MAX(last_analyze) FROM pg_stat_all_tables where schemaname=ANY(ARRAY[string_to_array('$schema_list', '|')]) GROUP BY schemaname;")
```

Comment:
Interesting, what's the motivation to do it here, rather than in the golang layer? 
is it because the user can run the assessment script directly?